### PR TITLE
Fix missing console log handler

### PR DESCRIPTION
**Pull request by eslickj**
This is an important fix.  I left the handler out of the new default config, so you won't see log messages by default.  This corrects the issue.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/config.py [hunk 1]
```diff
@@ -56,81 +56,6 @@
     "DYLD_LIBRARY_PATH": os.environ.get("DYLD_LIBRARY_PATH", ""),
 }
 
-# Default configuration json string.  Store the default config like this because
```
Comment (andrewlee94) [inline idaes/config.py:59]: Just to be sure, was all this deletion expected?

Comment (eslickj) [reply idaes/config.py:59]: Yeah.  I moved the defaults in the Pyomo config block.

**Review by Andresj89 — APPROVED** [review]
LGTM

**Review by jghouse88 — APPROVED** [review]
LGTM

**Pull request merged**